src/lib/output/themes/default/partials/icon.tsx

```diff
@@ -52,50 +52,88 @@ export const icons: Record<
 > = {
     [ReflectionKind.Accessor]: () =>
         kindIcon(
-            <path
-                d="M8.85 16L11.13 7.24H12.582L14.85 16H13.758L13.182 13.672H10.53L9.954 16H8.85ZM10.746 12.76H12.954L12.282 10.06C12.154 9.548 12.054 9.12 11.982 8.776C11.91 8.432 11.866 8.208 11.85 8.104C11.834 8.208 11.79 8.432 11.718 8.776C11.646 9.12 11.546 9.544 11.418 10.048L10.746 12.76Z"
-                fill="var(--color-text)"
-            />,
-            "#FF4D4D",
+            <text
+                fill="var(--color-icon-text)"
+                font-family="var(--icon-font-family)"
+                font-size="var(--icon-font-size)"
+                font-weight="var(--icon-font-weight)"
+                font-style="var(--icon-font-style)"
```

Comment:
I'm worried about exposing all of this with the existing setup. Exposing it this way may be convenient for the one or two people who just want to tweak this, but it implies that changing it will for at least some inputs result in reasonable output... which it won't.

Different font family = likely different width characters, so it won't be centered properly anymore, especially with the hardcoded x/y coordinates... even the existing set of font families likely won't quite play.
Different font size = Almost certainly won't fit properly.
Different font weight = Okay, this one is probably fine... but why? To expose everything? Making everything infinitely tweakable via CSS is very scary to me as essentially any change to the CSS is potentially breaking someone.
Different font style = Ditto...

Changing the text/background/border colors, sure, the rest I'm going to need some convincing for...

Reply:
Sure thing, my goal was to make sure people had the ability to adjust those settings to ensure they could be made as accessible as possible, but I can bring it back. We're doing a lot of things at work right now with accessibility practices, and I may have gone overboard.